frontend/src/components/visualizations/WSI/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "wsi-visualization",
```

Comment:
Can we rename this to something more iiif generic? Since any image can be handled this way?

Reply:
Yes, makes sense. I will change it to use IIIF instead.

Reply:
I'm also updating the visualization config MIME type to a custom one image/x-iiif. Please let me know if it needs to be kept as application/octet-stream.